impeller/renderer/backend/gles/command_buffer_gles.cc

```diff
@@ -40,7 +40,7 @@ bool CommandBufferGLES::OnSubmitCommands(CompletionCallback callback) {
 
 // |CommandBuffer|
 std::shared_ptr<RenderPass> CommandBufferGLES::OnCreateRenderPass(
-    RenderTarget target) const {
+    RenderTarget target) {
```

Comment:
@chinmaygarde  const here is gone since on vulkan we have to stash render-pass with the command buffer so it's not destroyed until the submission is complete.